src/card.css

```diff
@@ -0,0 +1,89 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+}
+  
+.card__img {
+  width: 160px;
+  height: 134px;
+  padding: 0 3px 36px ;
+}
+  
+.card__stars_box {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+}
+  
+.card__title {
+  font-size: 12px;
+  font-weight: 500;
+  color: #060b35;
+  height: 36px;
+  line-height: 18px;
+}
+  
+.card__code {
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  height: 14px;
+  margin-bottom: 16px;
+  margin-top: 4px;
```

Comment:
check point 10 in checklist
you shouldn't add both margin-top and margin-bottom to the elements
the logic should be the next: you add correct paddings to the whole card, then add all needed elements one by one, and specify correct margin-bottom for each element
if you'll have any questions about implementation, ask please in main channel for questions ❤️